tests/test_app/serializers.py

```diff
@@ -27,3 +26,32 @@ class CustomRevisionSerializer(RevisionSerializer):
 
 class CustomVersionSerializer(VersionSerializer):
     revision = CustomRevisionSerializer()
+
+
+class ParentTestModelSerializer(serializers.ModelSerializer):
```

Comment:
https://github.com/beda-software/drf-writable-nested
we could use this.... but it would add a dependency in the tests

Reply:
No, your solution is awesome!